detection-rules/impersonation_netflix.yml

```diff
@@ -14,6 +14,25 @@ source: |
     or strings.ilevenshtein(sender.display_name, 'netflix') <= 1
     or strings.ilike(sender.email.domain.domain, '*netflix*')
   )
+  and (
+    beta.whois(sender.email.domain).days_old < 90
+    or (
+      any(body.links,
+          .href_url.domain.tld not in ("com", "org", "net", "ms") and .href_url.domain.valid == true
+      )
+    )
+    or 2 of (
+      strings.icontains(body.current_thread.text, "restart"),
+      strings.icontains(body.current_thread.text, "cancelled"),
+      strings.icontains(body.current_thread.text, "membership"),
+      strings.icontains(body.current_thread.text, "expir"),
+      strings.icontains(body.current_thread.text, "payment")
+      and length(body.links) > 0
+      and not any(body.links,
+                  .href_url.domain.root_domain in ("netflix.com", "dvd.com", "netflixfamily.com")
```

Comment:
Do these checks belong in here (`2 of (...`?

(I don't think this is related the mimic tests failing, just something I noticed)

Reply:
Nice!! Thank you, yeah i think we're just missing a set of parens will update.